images/docker-cli/README.md

```diff
@@ -0,0 +1,28 @@
+<!--monopod:start-->
+# docker-cli
+| | |
+| - | - |
+| **OCI Reference** | `cgr.dev/chainguard/docker-cli` |
+
+
+* [View Image in Chainguard Academy](https://edu.chainguard.dev/chainguard/chainguard-images/reference/docker-cli/overview/)
+* [View Image Catalog](https://console.enforce.dev/images/catalog) for a full list of available tags.
+* [Contact Chainguard](https://www.chainguard.dev/chainguard-images) for enterprise support, SLAs, and access to older tags.*
+
+---
+<!--monopod:end-->
+
+<!--overview:start-->
+
+<!--overview:end-->
+
+<!--getting:start-->
+## Download this Image
+The image is available on `cgr.dev`:
+
+```
+docker pull cgr.dev/chainguard/docker-cli:latest
+```
+<!--getting:end-->
+
```

Comment:
Is there anything more we can add to the readme?

Reply:
I may or may not have forgotten about the README 👀

Reply:
Updated!